src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -90,13 +90,19 @@ class JesBackendCall(val backend: JesBackend,
 
   override def execute(implicit ec: ExecutionContext) = backend.execute(this)
 
+  def pollingLog(runId: String, status: RunStatus) = logger.info(s"Polled JES Job $runId. Job status : $status")
+
   override def poll(previous: ExecutionHandle)(implicit ec: ExecutionContext) = Future {
     previous match {
       case handle: JesPendingExecutionHandle =>
         val status = Try(handle.run.checkStatus(this, handle.previousStatus))
```

Comment:
shouldn't the logging be right above this line where we actually do the status check?